vendor/bat-native-ledger/include/bat/ledger/ledger.h

```diff
@@ -279,6 +281,11 @@ class LEDGER_EXPORT Ledger {
       const ledger::MonthlyStatementCallback& callback,
       ledger::ACTIVITY_MONTH month,
       uint32_t year) = 0;
+
+  virtual void GetOneTimeTipsStatements(
```

Comment:
I think we should just add an optional month and year to the existing GetOneTimeTips method